task/host-sp-comms/src/main.rs

```diff
@@ -840,8 +847,36 @@ impl ServerImpl {
                 Some(SpToHost::Alert { action: 0 })
             }
             HostToSp::RotRequest => {
-                // TODO forward request to RoT
-                Some(SpToHost::RotResponse)
+                match attest_data::messages::parse_message(data) {
+                    Ok((command, data)) => {
+                        let n = usize::min(MAX_DATA_LEN, data.len());
+
+                        let mut data_buf: [u8; MAX_DATA_LEN] =
+                            [0; MAX_DATA_LEN];
```

Comment:
Is `MAX_DATA_LEN` small enough that putting it on the stack is okay? Or should we statically allocate space up front? (Worried about a stack overflow that we only hit with certain messages)

Reply:
`MAX_DATA_LEN` is currently the size of a nonce which is 32 bytes (https://github.com/oxidecomputer/dice-util/pull/220/files#diff-d7aca782c921cfedc9fafda8a7258ce09c947b295ed7457bfc0956e0bbaab82dR12) . I'm expecting this to grow to 64 bytes for signing a hash but not more than that.